bundle/manifests/siteconfig-cluster-instance-webhook-service_v1_service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: siteconfig-webhook-server-cert
+  creationTimestamp: null
+  labels:
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: siteconfig
+  name: siteconfig-cluster-instance-webhook-service
```

Comment:
This service name doesn't match the one in the webhook [here](https://github.com/stolostron/siteconfig/pull/207/files#diff-369b61dd1f2f1f60722927fda70e5a51e130cea68336e6d536086b522ffef0c6R13)

I'd say do a regenerate and a quick retest to make sure this is working.

Reply:
Kustomize was configured to add a `siteconfig-` prefix. Removed this and added the prefix to all the related resources.